CMakeLists.txt

```diff
@@ -0,0 +1,441 @@
+cmake_minimum_required(VERSION 3.3.0 FATAL_ERROR)
+project(NEURON C CXX)
```

Comment:
CMake project name is inconsistent with the repository name, and most CMake variable prefixed with `NRN_`.
I suggest to use NRN everywhere.

Reply:
created separated issue : https://github.com/neuronsimulator/nrn/issues/355